sdk/Pulumi.Automation/Workspace.cs

```diff
@@ -407,6 +407,14 @@ public Task InstallPluginAsync(string name, string version, PluginKind kind, Can
         /// <param name="cancellationToken">A cancellation token.</param>
         public abstract Task ChangeSecretsProviderAsync(string stackName, string newSecretsProvider, SecretsProviderOptions? secretsProviderOptions = null, CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Manually install packages and plugins required by your program or policy pack
```

Comment:
Manually? That feels like the wrong description for this.

Reply:
Good catch - I think I pulled this from an older version of the Cli. I've updated to reflect the working in the current Cli help for the command.